Source/Evolve/WarpXEvolve.cpp

```diff
@@ -415,6 +417,8 @@ WarpX::OneStep_sub1 (Real curtime)
     doQEDEvents();
 #endif
 
+    mypc->doResampling(istep[0]+1);
```

Comment:
Same as above. I would suggest to add a short comment to clarify why we need to pass `istep[0] + 1` instead of just`istep[0]`